docs/source/index.rst

```diff
@@ -57,10 +63,18 @@ PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
    torch.utils.dlpack <dlpack>
    torch.utils.model_zoo <model_zoo>
    torch.utils.tensorboard <tensorboard>
-   type_info
+   torch.__config__ <__config__>
```

Comment:
The `torch.__config__` docs on master don't seem to point anywhere...

Reply:
Yeah I've never been sure what that's supposed to point to. Do you know?

Reply:
I'm not sure. At any rate it's unrelated to this change so let's not worry too much about it